docs/core-tags.md

```diff
@@ -206,14 +206,14 @@ _HTML Output:_
 
 ## `body-only-if`
```

Comment:
I think we should move this whole section (sans the `body-only-if` heading) into a pro tip on the dynamic tag name syntax section: https://github.com/marko-js/marko/blob/master/docs/syntax.md#dynamic-tagname

Reply:
Moved code